wsgi_intercept/_urllib3.py

```diff
@@ -30,6 +30,9 @@ def __init__(self, *args, **kwargs):
             kwargs.pop('socket_options', None)
             kwargs.pop('key_password', None)
             kwargs.pop('server_hostname', None)
+            if sys.version_info > (3, 12):
+                self.key_file = kwargs.pop('key_file', None)
+                self.cert_file = kwargs.pop('cert_file', None)
```

Comment:
What's the assignment to self.key_file and self.cert_file accomplishing here? In the existing tests it's not required.

If you feel like it is required can you add a test that would fail without it, so the reasoning is clear?

Reply:
Indeed, it is possible without assignment